### PR TITLE
added image

### PR DESCRIPTION
**Pull request by IhtishamMushtaq**
modified image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <script src="index.js"></script>
     <title>Document</title>
 </head>
+
 <body>
     <h1>Intro to Git Branching</h1>
+    <center><img src="download.jpg" alt="image"></center>>
```
Comment (Salman18) [inline index.html:14]: why are using center tag

Comment (IhtishamMushtaq) [reply index.html:14]: to set the content at center

Comment (taheer786) [reply index.html:14]: ?????

File: index.html [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <script src="index.js"></script>
     <title>Document</title>
 </head>
+
 <body>
     <h1>Intro to Git Branching</h1>
```
Comment (Salman18) [inline index.html]: change to JS message in `h1`

**Pull request merged**